viam-orb-slam3.go

```diff
@@ -30,25 +29,29 @@ import (
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/registry"
 	"go.viam.com/rdk/resource"
-	"go.viam.com/rdk/rimage"
 	"go.viam.com/rdk/rimage/transform"
 	"go.viam.com/rdk/services/slam"
 	"go.viam.com/rdk/spatialmath"
 	rdkutils "go.viam.com/rdk/utils"
 	"go.viam.com/rdk/vision"
 	slamConfig "go.viam.com/slam/config"
 	"go.viam.com/slam/dataprocess"
+	slamSensorUtils "go.viam.com/slam/sensors/utils"
 	slamUtils "go.viam.com/slam/utils"
 	goutils "go.viam.com/utils"
 	"go.viam.com/utils/pexec"
 	"go.viam.com/utils/protoutils"
+	"golang.org/x/exp/slices"
 
 	"github.com/viamrobotics/viam-orb-slam3/internal/grpchelper"
 )
 
 var (
 	cameraValidationMaxTimeoutSec = 30 // reconfigurable for testing
 	dialMaxTimeoutSec             = 30 // reconfigurable for testing
+	Model                         = resource.NewModel("viam", "slam", "orbslamv3")
+	BinaryLocation                = "orb_grpc_server"
```

Comment:
[nit] The linter will complain that you didn't add a comment above these exported variables. I recommend merging the linter PR first and this PR second to ensure that your code won't fail the linter.

Reply:
Agreed.